### DISCUSSION TITLE
How to set monitor refresh rate in fullscreen mode?

### DISCUSSION THREAD

**Discussion by Webfra** [vsg-dev/VulkanSceneGraph#732] [Q&A]
Sorry for this probably very noobish question...

I'm experimenting with the vsgimgui_example.
I was able to change vsync setting and fullscreen mode via the windowTraits object.
But how do I change the refresh rate of the monitor to a value different from the default 60Hz?
(My graphics card and monitor allow refresh rates between 24Hz and 165Hz.)

Any pointers in the right direction would be appreciated.

**Comment by robertosfield** [comment]
The VSG doesn't set the refresh rate of a monitor, it'll just use what the windowing system is setup for. so to set the refresh rate you'll need to look at setting this directly via the windowing system APIs.
